docs/design/operator.md

```diff
@@ -243,7 +243,7 @@ status:
   reason: operator config successfully created
 ```
 
-### CephCSIDriver CRD
+### Driver CRD
 
 Manages the installation, lifecycle management, and configuration for CephFS,
 RBD, and NFS CSI drivers within namespaces.
```

Comment:
RBD and NFS CSI drivers within namespaces.
( Not related to current PR changes but still)

Reply:
You can ignore this comment. Not related to this PR.